content/docs/guides/proxy-exporter.md

```diff
@@ -0,0 +1,375 @@
+---
+title: Understanding and using the proxy exporter pattern
+---
+
+# Understanding and using the proxy exporter pattern
+
+Exporters that follow the proxy exporter pattern are those who get their target from the Prometheus GET request and scrape these targets while running on a different machine via a network. In this guide, we will:
+
+* describe the proxy exporter pattern and why it is used,
+* run the [blackbox](https://github.com/prometheus/blackbox_exporter) exporter as an example of the pattern,
+* let the blackbox exporter run basic metric queries against the Prometheus [website](https://prometheus.io),
+* configure a custom query module for the blackbox exporter,
+* examine a popular pattern of configuring Prometheus to scrape exporters using relabeling.
+
+## The proxy exporter pattern?
+
+Any exporter receives requests from Prometheus and answers them with data forwarded from other systems. So technically every [exporter](/docs/instrumenting/exporters/) is a [proxy](https://en.wikipedia.org/wiki/Proxy_server). But by proxy exporter pattern we refer to a specific design, in which:
+
+* the exporter will get the target’s metrics via a network protocol.
+* the exporter does not have to run on the machine the metrics are taken from.
+* the exporter gets the targets and a query config string as parameters of Prometheus’ GET request.
+* the exporter subsequently starts the scrape after getting Prometheus’ GET requests and once it is done with scraping.
+
+This pattern is only used for certain exporters, namely the [blackbox](https://github.com/prometheus/blackbox_exporter), the [SNMP](https://github.com/prometheus/snmp_exporter) and the [IPMI exporter](https://github.com/soundcloud/ipmi_exporter).
```

Comment:
IPMI is mixed now, best not to mention it

Reply:
Oh, I did not know that. Thanks for the hint.

Reply:
If IPMI is so confusing that it should not be part of the docs, that says a lot about IPMI exporter.

Reply:
It's not confusing just a bit unusual, but would cause confusion in an introductory guide like this as it is both types of exporter.